source/Directory.Build.props

```diff
@@ -0,0 +1,21 @@
+<Project>
+
+	<PropertyGroup>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<NoWarn />
+		<WarningsAsErrors />
+		<WarningsNotAsErrors />
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)' == 'Debug'">
```

Comment:
Do we have to define this explicitly or is it the default setting and you added this just for clarification?

Reply:
No - we don't have to define the _property_ `TreatWarningsAsErrors`, which has default _value_ `false`.
Added for clarity.